source/index.txt

```diff
@@ -37,9 +37,33 @@ specify callbacks to access them when communicating with MongoDB.
    For the main MongoDB documentation, see the :manual:`MongoDB Manual
    </>`.
 
+Learn
+-----
+
+Visit the Developer Hub and MongoDB University for resources related to
+the Node.js driver.
+
+Developer Hub
+~~~~~~~~~~~~~
+
+The Developer Hub contains developer related content, resources,
+community, and social engagement. 
+
+To view articles on Node.js, see the `How To's and Articles page
```

Comment:
Nitpick/not required for LGTM:

Would https://developer.mongodb.com/learn/?content=Articles&text=Node.js be a better link? That way readers will be directed to only articles that contain info on Node.js, rather than articles on javascript (because JavaScript could mean articles on triggers, functions, browser stuff, as well as Node.js driver stuff)